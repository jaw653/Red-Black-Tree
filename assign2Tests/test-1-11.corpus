"o	f jmv	
bs"
"miamb	

 hj"
"
 y
u	st 	z"
" 	d	g
t
as
"
"ndlt  s tyr"
"k m vsocyvr"
"b ep
mollqe"
"v	zdyodxucb"
"
azk	 
se h"
"lf
z	vac	de"
"
t 	fy	fkxg"
"
g
jqt
y
rf"
"hk sy		t	zy"
"fm	 z q	es
"
"gp
 f
rf
 
"
"l	wmduq 	
k"
"cnva	r	vf s"
"l
h		jq  de"
"ulyur	neu e"
"
pxenyu		 	"
"rqnsfjv
kta"
"mntd	amfuwq"
"p	lcu 
bjb
"
"gx f	gc	 he"
"

b
iej wtj"
"
		hf
z
d
o"
" ntaba	
a
i"
" jpq	 b
zxp"
"
ttls	tle	o"
"wp zu l ekw"
"tn	akjvmi g"
"it
z 
 cth
"
" mlp	
a
iv
"
"xp pdksky	 "
"u 	bhymcfvd"
"guev
a	b 	 "
"igetf b jpf"
"


h	k	p
hl"
"tnk
x

  f "
"q	qs
jm 
f	"
"up

i
ftg o"
"	
ysj
ir
pp"
"n ad	j z		h"
" t li uajmq"
" lwt vi bzm"
"
lnqc s 
uw"
"qkfhax
qu	k"
"q
 n
	
lqh	"
"
wmt	 	n zf"
"a
i		
gap
n"
" sr
ugb qhh"
"e mgiekq   "
"jtgpsrmt	
z"
"xqkl	pnm	f
"
"zrlty	 dco	"
"udlz	gdwi	
"
"nl
qa sj w "
"

	gks a	pn"
"gjwaihteun "
" qhcv	bq ot"
"	awjjczjpd "
"s
  hxby 
s"
"l 
	 xjdc
v"
" ggwb	cieyi"
"sruimnnqn	j"
"b	qnbw	ql	e"
"	thibzco xg"
"qwu	swg
 es"
"ls inkgah	x"
"sqpri
 f	ac"
"m 	p
w

atv"
"x	hi  m	qxl"
"yri l
bydio"
" uwsei
 	
 "
" 
cgq
 	  d"
"lf	zjnav
hq"
"

	e	r wgj "
"
s
	rc qdeh"
"h 	
 xc				"
"m
 u
hr ce	"
"l	mn	r 
w	z"
"s 	di
mthy	"
"
l	e
	d  nx"
"xt	
tcmo
d
"
"  nua ptgpj"
"gb	mps	qi a"
"wa	 qn
nmnb"
"g y
 
zvl
 "
"	 jw jhtdew"
"
 
f 
v
f	q"
"zw cadu	qkp"
"ntep 
dx x
"
"atvi	u
omm
"
"	plh vy	x
	"
"kf
wg	pt	k
"
"flgy	xn isw"
"ki			 o	
cv"
"h  suqlh rx"
"
ar

	
l
tl"
"
n	
aoc
huc"
" t
d	iqt  o"
"

gy z	vqhw"
"orfib	  	 k"
"gqo
 v	gluu"
"mc
n	od
	nq"
"	ycv
 umsv
"
"  ed
wladsr"
"fgwum		c
fl"
"qpas
zdk i
"
"
s	ppde

yc"
"j	 w	 as
tw"
"	
izor			 r"
"	dfgs
tq	j
"
"sox	h
mvn j"
"r		

xiuc	
"
"	eni	  urwe"
"qr	bj s yaf"
"
m irir s	f"
"j 
i
ga
jt	"
"b ok	

		hb"
"m cn	k zwhy"
"ze 
ypk  
r"
"ky	yk	rl	wa"
"bwehg
si mj"
