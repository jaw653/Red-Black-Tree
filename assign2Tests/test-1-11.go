s
r
d "o	f jmv	
bs"
d "miamb	

 hj"
d "
 y
u	st 	z"
d " 	d	g
t
as
"
d "k m vsocyvr"
d "v	zdyodxucb"
d "
azk	 
se h"
d "lf
z	vac	de"
d "
t 	fy	fkxg"
d "
g
jqt
y
rf"
d "hk sy		t	zy"
d "fm	 z q	es
"
d "l	wmduq 	
k"
d "cnva	r	vf s"
d "l
h		jq  de"
d "ulyur	neu e"
d "
pxenyu		 	"
d "rqnsfjv
kta"
d "p	lcu 
bjb
"
d "gx f	gc	 he"
d "

b
iej wtj"
d "
		hf
z
d
o"
d " jpq	 b
zxp"
d "
ttls	tle	o"
d "wp zu l ekw"
d "tn	akjvmi g"
d "it
z 
 cth
"
d " mlp	
a
iv
"
d "xp pdksky	 "
d "guev
a	b 	 "
d "igetf b jpf"
d "tnk
x

  f "
d "q	qs
jm 
f	"
d "up

i
ftg o"
d "	
ysj
ir
pp"
d "n ad	j z		h"
d " t li uajmq"
d " lwt vi bzm"
d "
lnqc s 
uw"
d "qkfhax
qu	k"
d "q
 n
	
lqh	"
d "
wmt	 	n zf"
d "a
i		
gap
n"
d " sr
ugb qhh"
d "e mgiekq   "
d "jtgpsrmt	
z"
d "xqkl	pnm	f
"
d "udlz	gdwi	
"
d "nl
qa sj w "
d "

	gks a	pn"
d "gjwaihteun "
d " qhcv	bq ot"
d "	awjjczjpd "
d "s
  hxby 
s"
d "l 
	 xjdc
v"
d " ggwb	cieyi"
d "sruimnnqn	j"
d "b	qnbw	ql	e"
d "ls inkgah	x"
d "sqpri
 f	ac"
d "m 	p
w

atv"
d "x	hi  m	qxl"
d " uwsei
 	
 "
d " 
cgq
 	  d"
d "lf	zjnav
hq"
d "
s
	rc qdeh"
d "h 	
 xc				"
d "m
 u
hr ce	"
d "l	mn	r 
w	z"
d "
l	e
	d  nx"
d "xt	
tcmo
d
"
d "  nua ptgpj"
d "gb	mps	qi a"
d "wa	 qn
nmnb"
d "g y
 
zvl
 "
d "	 jw jhtdew"
d "
 
f 
v
f	q"
d "zw cadu	qkp"
d "ntep 
dx x
"
d "atvi	u
omm
"
d "	plh vy	x
	"
d "kf
wg	pt	k
"
d "flgy	xn isw"
d "ki			 o	
cv"
d "h  suqlh rx"
d "
ar

	
l
tl"
d "
n	
aoc
huc"
d " t
d	iqt  o"
d "

gy z	vqhw"
d "orfib	  	 k"
d "gqo
 v	gluu"
d "mc
n	od
	nq"
d "	ycv
 umsv
"
d "  ed
wladsr"
d "qpas
zdk i
"
d "
s	ppde

yc"
d "	
izor			 r"
d "	dfgs
tq	j
"
d "sox	h
mvn j"
d "r		

xiuc	
"
d "	eni	  urwe"
d "
m irir s	f"
d "j 
i
ga
jt	"
d "b ok	

		hb"
d "m cn	k zwhy"
d "ze 
ypk  
r"
d "ky	yk	rl	wa"
d "bwehg
si mj"
s
r
