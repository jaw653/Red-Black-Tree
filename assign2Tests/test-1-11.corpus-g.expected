0: o f jmv bs(o f jmv bs)-
1: miamb hj(o f jmv bs)-l y u st z(o f jmv bs)-r
2: d g t as(miamb hj)-l ndlt s tyr(miamb hj)-r v zdyodxucb(y u st z)-l ysj ir pp(y u st z)-r
3: b ep mollqe(d g t as)-l k m vsocyvr(d g t as)-r mntd amfuwq(ndlt s tyr)-l ntaba a i(ndlt s tyr)-r t fy fkxg(v zdyodxucb)-l wp zu l ekw(v zdyodxucb)-r yri l bydio(ysj ir pp)-l zrlty dco(ysj ir pp)-r
4: azk se h(b ep mollqe)-l cnva r vf s(b ep mollqe)-r g jqt y rf(k m vsocyvr)-l lf z vac de(k m vsocyvr)-r =mlp a iv(mntd amfuwq)-l n ad j z h(mntd amfuwq)-r =nl qa sj w(ntaba a i)-l nua ptgpj(ntaba a i)-r pxenyu(t fy fkxg)-l ulyur neu e(t fy fkxg)-r wmt n zf(wp zu l ekw)-l xp pdksky(wp zu l ekw)-r =ycv umsv(yri l bydio)-l =ze ypk r(zrlty dco)-l =zw cadu qkp(zrlty dco)-r
5: a i gap n(azk se h)-l b iej wtj(cnva r vf s)-l fm z q es(g jqt y rf)-l hk sy t zy(g jqt y rf)-r l wmduq k(lf z vac de)-l lwt vi bzm(lf z vac de)-r =n aoc huc(n ad j z h)-r =ntep dx x(nua ptgpj)-l p lcu bjb(pxenyu)-l rqnsfjv kta(pxenyu)-r ttls tle o(ulyur neu e)-l up i ftg o(ulyur neu e)-r =wa qn nmnb(wmt n zf)-l =x hi m qxl(xp pdksky)-l xqkl pnm f(xp pdksky)-r
6: awjjczjpd(a i gap n)-r b qnbw ql e(b iej wtj)-r e mgiekq(fm z q es)-l gp f rf(hk sy t zy)-l jpq b zxp(hk sy t zy)-r l h jq de(l wmduq k)-l =l xjdc v(l wmduq k)-r lnqc s uw(lwt vi bzm)-l m p w atv(lwt vi bzm)-r =orfib k(p lcu bjb)-l =plh vy x(p lcu bjb)-r q qs jm f(rqnsfjv kta)-l sr ugb qhh(rqnsfjv kta)-r tn akjvmi g(ttls tle o)-l u bhymcfvd(ttls tle o)-r =uwsei(up i ftg o)-r =xt tcmo d(xqkl pnm f)-r
7: atvi u omm(awjjczjpd)-l =b ok hb(b qnbw ql e)-l cgq d(b qnbw ql e)-r =dfgs tq j(e mgiekq)-l e r wgj(e mgiekq)-r gks a pn(gp f rf)-l gx f gc he(gp f rf)-r it z cth(jpq b zxp)-l jtgpsrmt z(jpq b zxp)-r l e d nx(l h jq de)-l =l mn r w z(l h jq de)-r =lf zjnav hq(lnqc s uw)-l =ls inkgah x(lnqc s uw)-r m irir s f(m p w atv)-l m u hr ce(m p w atv)-r =q n lqh(q qs jm f)-l qkfhax qu k(q qs jm f)-r s hxby s(sr ugb qhh)-l sruimnnqn j(sr ugb qhh)-r t li uajmq(tn akjvmi g)-l =tnk x f(tn akjvmi g)-r =udlz gdwi(u bhymcfvd)-r
8: =ar l tl(atvi u omm)-l =bwehg si mj(cgq d)-l f v f q(e r wgj)-r gjwaihteun(gks a pn)-l guev a b(gx f gc he)-l hf z d o(gx f gc he)-r =igetf b jpf(it z cth)-l j w as tw(it z cth)-r =jw jhtdew(jtgpsrmt z)-r kf wg pt k(l e d nx)-l =m cn k zwhy(m irir s f)-l =mc n od nq(m u hr ce)-r =qhcv bq ot(qkfhax qu k)-l qwu swg es(qkfhax qu k)-r =s di mthy(s hxby s)-l sqpri f ac(s hxby s)-r =t d iqt o(sruimnnqn j)-r =thibzco xg(t li uajmq)-r
9: ed wladsr(f v f q)-l flgy xn isw(f v f q)-r ggwb cieyi(gjwaihteun)-l =gqo v gluu(guev a b)-l h k p hl(hf z d o)-l izor r(j w as tw)-l ki o cv(kf wg pt k)-r qpas zdk i(qwu swg es)-l =r xiuc(qwu swg es)-r s rc qdeh(sqpri f ac)-l
10: =eni urwe(ed wladsr)-r =fgwum c fl(flgy xn isw)-l gb mps qi a(ggwb cieyi)-l =gy z vqhw(h k p hl)-l h xc(h k p hl)-r =j i ga jt(izor r)-r =ky yk rl wa(ki o cv)-r =qr bj s yaf(qpas zdk i)-r =s ppde yc(s rc qdeh)-l =sox h mvn j(s rc qdeh)-r
11: =g y zvl(gb mps qi a)-l =h suqlh rx(h xc)-l
Words/Phrases: 124
Nodes: 124
Minimum depth: 4
Maximum depth: 12
0: ntaba a i(ntaba a i)-
1: j w as tw(ntaba a i)-l u bhymcfvd(ntaba a i)-r
2: b ep mollqe(j w as tw)-l ndlt s tyr(j w as tw)-r thibzco xg(u bhymcfvd)-l yri l bydio(u bhymcfvd)-r
3: h k p hl(b ep mollqe)-r =mntd amfuwq(ndlt s tyr)-l s di mthy(thibzco xg)-l =zrlty dco(yri l bydio)-r
4: fgwum c fl(h k p hl)-l qr bj s yaf(s di mthy)-l
5: =e r wgj(fgwum c fl)-l =gp f rf(fgwum c fl)-r =qwu swg es(qr bj s yaf)-r
Words/Phrases: 16
Nodes: 16
Minimum depth: 3
Maximum depth: 6
